benchmarks/benchmark_filters.py

```diff
@@ -0,0 +1,17 @@
+# See "Writing benchmarks" in the asv docs for more information.
+# http://asv.readthedocs.io/en/latest/writing_benchmarks.html
+import numpy as np
+from skimage import filters
+
+
+class FiltersSuite:
+    """Benchmark for filter routines in scikit-image."""
+    def setup(self):
+        self.image = np.random.random((4000, 4000))
+        self.image[:2000, :2000] += 1
+        self.image[3000:, 3000] += 0.5
```

Comment:
@jni Should we stay within our dtype standards (i.e. range [-1|0, 1]) ? :)

Reply:
I don't think this matters.

Reply:
to be more precise: certainly it doesn't matter in the benchmarks. As to the library as a whole, I really want it to be friendlier to "out-of-range" images, so I don't mind not adhering to that standard where we don't have to. Of course, the overall messaging will need to change.